docs/community/governance.md

```diff
@@ -0,0 +1,180 @@
+---
+title: "Governance"
+---
+
+# Governance
+
+This document describes the rules and governance of the project. It is a slightly modified version of the [Prometheus Governance](https://prometheus.io/governance/#governance-changes) and similar to the [Thanos Governance](https://thanos.io/tip/thanos/governance.md/)
+
+It is meant to be followed by all the developers of the Observatorium project and the Observatorium community. Common terminology used in this governance document are listed below:
+
+* **Maintainers Team**: A core Observatorium team that have owner access to http://github.com/observatorium organization and all projects within it. Current list is available [here][maintainers-doc].
+
+* **The Observatorium project**: The sum of all activities performed under the [Observatorium organization on GitHub][gh], concerning one or more repositories or the community.
+
+Both Triage and Maintainers have are part of [`observatorium-team@googlegroups.com`][team] email list.
+
+## Values
+
+The Observatorium developers and community are expected to follow the values defined in the [CNCF charter][charter], including the [CNCF Code of Conduct][coc].
+Furthermore, the Observatorium community strives for kindness, giving feedback effectively, and building a welcoming environment. The Observatorium developers generally decide by consensus and only resort to conflict resolution by a majority vote if consensus cannot be reached.
+
+## Decision making
+
+### Maintainers Team
+
+Team member status may be given to those who have made ongoing contributions to the Observatorium project for at least 3 months.
+This is usually in the form of code improvements and/or notable work on documentation, but organizing events or user support could also be taken into account.
+
+New members may be proposed by any existing Maintainer by email to [Observatorium][team]. It is highly desirable to reach consensus about acceptance of a new member.
```

Comment:
```suggestion
New members may be proposed by any existing Maintainer by sending an email to the [Observatorium][team]. It is highly desirable to reach consensus about acceptance of a new member.
```

Reply:
What do we mean here with the note about reaching consensus?

Reply:
100% vote